apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+	type Path,
+	type PathValue,
+} from "react-hook-form";
+import ReactSelect from "react-select";
+
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useFormController } from "~/libs/hooks/hooks.js";
+import { type SelectOption } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties<TFieldValues extends FieldValues, TOptionValue> = {
+	control: Control<TFieldValues, null>;
+	isMulti?: boolean;
+	label: string;
+	name: FieldPath<TFieldValues>;
+	options: SelectOption<TOptionValue>[];
+};
+
+const Select = <TFieldValues extends FieldValues, TOptionValue>({
+	control,
+	isMulti = false,
+	label,
+	name,
+	options,
+}: Properties<TFieldValues, TOptionValue>): JSX.Element => {
+	const { field } = useFormController({
+		control,
+		name,
+	});
+
+	return (
+		<label className={styles["label"]}>
+			<span className={styles["label-text"]}>{label}</span>
```

Comment:
Not sure, if this [comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/50#discussion_r1727521295) was fully done. For example, I will need to hide label for select in pagination component, but for now your code does not provide the ability to hide the label.
So I suggest to add additional prop `isLabelVissible`, which should be `true` by default, and depending on value of this prop, you will choose class for this element: `className={isLabelVissible? styles["label-text"] : "visually-hidden"}`

Reply:
"But for now it's not needed"

Reply:
Okay, it makes sense. By the way, I'll be able to add this "feature", in my PR, I guess